benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -243,13 +360,31 @@ def run_benchmark(rank, model, data, args, config):
 
 
 def get_json_config(file_name, id):
-    with open(os.path.join(Path(__file__).parent, file_name), "r") as f:
```

Comment:
I'll add this back when responding to comments. I didn't realize someone submitted a pr on the benchmark.

Reply:
I fixed it.